mcs_benchmark_data/pipelines/commonsenseQaBenchmark_pipeline.py

```diff
@@ -0,0 +1,73 @@
+from typing import Optional, List, Tuple
+
+from rdflib import URIRef
+
+from mcs_benchmark_data._pipeline import _Pipeline
+from mcs_benchmark_data._loader import _Loader
+from mcs_benchmark_data.extractors.nop_extractor import NopExtractor
+from mcs_benchmark_data.transformers.CommonsenseQa_transformer import (
+    CommonsenseQaTransformer
+)
+
+
+class CommonsenseQaBenchmarkPipeline(_Pipeline):
+    def __init__(
+        self,
+        *,
+        id: str,

```

Comment:
id is used to know where to put `loaded` files. This can be hard-coded e.g., `commonsense_qa_benchmark` or `data/benchmark/commonsense_qa` or whatever.